packages/firestore/src/platform_browser/webchannel_connection.ts

```diff
@@ -244,8 +244,7 @@ export class WebChannelConnection implements Connection {
     // ReactNative and so we exclude it, which just means ReactNative may be
     // subject to the extra network roundtrip for CORS preflight.
     if (!isReactNative()) {
-      // @ts-ignore
-      request['httpHeadersOverwriteParam'] = '$httpHeaders';
+      (request as Indexable)['httpHeadersOverwriteParam'] = '$httpHeaders';
```

Comment:
I wasted more time on this then I should have, but here you go:

```
interface WebChannelOptions {
  messageHeaders?:StringMap;  
  initMessageHeaders?:StringMap;
  messageContentType?:string;
  messageUrlParam?:StringMap;
  clientProtocolHeaderRequired?:boolean;
  concurrentRequestLimit?:number;
  supportsCrossDomainXhr?:boolean ;
  testUrl?:string;
  sendRawJson?:boolean;
  httpSessionIdParam?:string;
  httpHeadersOverwriteParam?:string;
  backgroundChannelTest?:boolean ;
  forceLongPolling?:boolean;
  fastHandshake?:boolean;
  disableRedac?:boolean ;
  clientProfile?:string ;
  internalChannelParams:{[key:string]:boolean|number} ;
  xmlHttpFactory:unknown,
  requestRefreshThresholds:{[key:string]:number},
}

const request: WebChannelOptions;
```

This is baed on https://cs.corp.google.com/piper///depot/google3/javascript/closure/labs/net/webchannel.js?rcl=242544058&l=196

Reply:
Thanks! Added.